crowbar_framework/app/controllers/api/ceph_controller.rb

```diff
@@ -14,23 +14,22 @@
 # limitations under the License.
 #
 
-class Api::StoragesController < ApiController
+class Api::CephController < ApiController
   before_action :set_ceph
 
-  api :GET, "/api/storages", "List all Ceph storages"
+  api :GET, "/api/ceph", "Show the Ceph object"
```

Comment:
I agree that something less vague would be better. What does the API call return, exactly? Ideally the description should clearly reflect that.